src/Catel.Core/Catel.Core.Shared/Data/ModelBase.propertyediting.cs

```diff
@@ -122,7 +122,7 @@ internal void SetValue(PropertyData property, object value, bool notifyOnChange,
                 }
 
                 // Validate before assigning, dynamic properties will cause exception
-                if (validateAttributes && !LeanAndMeanModel)
+                if (validateAttributes && !LeanAndMeanModel && ValidateUsingDataAnnotations)
```

Comment:
I already thought about this: _before_ the "SuspendValidation" check to add a "ValidateUsingDataAnnotations" check to supress the adding of the property name to the HashSet. But "Validate()" has a "validateDataAnnotations" parameter which I thought I had to pass through - imho this would get it a bit too complex.

But if you do not think so and we do not need to pass through the parameter postbone the check and do this:

    private bool ValidatePropertyUsingAnnotations(string propertyName, object value, PropertyData catelPropertyData)
    {
        if (ValidateUsingDataAnnotations)
        {
            return true;
        }
        
        if (SuspendValidation)
        {
    [...]